modules/mono/config.py

```diff
@@ -93,7 +97,10 @@ def configure(env):
                 env.Append(LINKFLAGS='LIBCMT' + lib_suffix)
                 env.Append(LINKFLAGS='Psapi' + lib_suffix)
             else:
-                env.Append(LIBS=mono_static_lib_name)
+                env.Append(LINKFLAGS=os.path.join(mono_lib_path, mono_static_lib_name + lib_suffix))
```

Comment:
Is using LINKFLAGS here needed? Can it be done by setting `env.Append(LIBPATH=mono_lib_path)`?

Reply:
Probably. I'll check. If that works I'll make a new pr